src/GovUk.Education.ExploreEducationStatistics.Data.Api/Services/PermalinkService.cs

```diff
@@ -1,44 +1,83 @@
-using System;
-using System.Threading.Tasks;
-using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
-using GovUk.Education.ExploreEducationStatistics.Data.Api.Models;
-using GovUk.Education.ExploreEducationStatistics.Data.Api.Services.Interfaces;
-using Microsoft.Azure.Cosmos.Table;
-
 namespace GovUk.Education.ExploreEducationStatistics.Data.Api.Services
 {
+    using System;
```

Comment:
We've got the `using` imports outside of the namespace in every other file. Rider will want to move these outside if I organise imports on this file. Also many of the imports are unused.